bosh-audit.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
```

Comment:
Could we add an overview docstring with the description from the PR here?

Reply:
A description is in the ArgParser description but would be nice to overview for future devs

Reply:
that seems smart

Reply:
Can you append that to the argparse help? Since you're at it...

Reply:
Done